config/pod.yaml.hbs

```diff
@@ -21,9 +21,11 @@ spec:
       privileged: {{privileged}}
     resources:
       limits:
-        cpu: {{cpu}}m
+        cpu: {{cpu}}
```

Comment:
what was `m` doing before ?

Reply:
nothing

Reply:
You are correct. we use `m` apparently didn't see declaration in default.yaml (0.5), which is used in one of our cluster.